curriculum/js1/5.js

```diff
@@ -5,8 +5,12 @@
  * @param {function} b
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (X, fx) => {
```

Comment:
you should never start a variable with a capitalized character unless its a CONSTANT